src/utils/getNormalizedStateData.js

```diff
@@ -72,7 +72,7 @@ const getNormalizedStateData = (data, stateCode) => {
           month: toInt(monthReported - 1),
           day: toInt(dayReported),
         },
-        value: parseFloat(item.value),
+        value: toInt(item.value),
```

Comment:
Why did we make this switch?

Reply:
Oh, this and below question comes from previous branch, didn't notice that. Fixed.